### PR TITLE
Add ONT setup details to README

### PR DESCRIPTION
**Pull request by avarsava**
Run Scanner User Manual is in progress, in the meantime, specify unique
settings required by nanopore sequencers in README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -70,8 +77,8 @@ The parameters are set based on the processor.
 - PACBIO/default requires `address` to be set to the URL of the PacBio machine.
 - ILLUMINA/default optionally allows `checkOutput`. If true, the scanner will
   try to look for BCL files to verify a run is complete if no logs are present.
-  If false, it will assume the run is complete if ambiguous. The default is true.
-  This can be very slow on certain network file systems.
+  If false, it will assume the run is complete if ambiguous. The default is true. This can be very slow on certain network file systems.
+- OXFORDNANOPORE processors require "name" to be set to either "minion" or "promethion" based on model. Additionally, a unique "name" for the sequencer must be specified in the "parameters".
```
Comment (apmasell) [inline README.md]: Use backticks to match the style of the others.

Comment (callunity) [inline README.md]: For consistency with the bullet points above, could the words in quotes be changed to words in backticks? "OXFORDNANOPORE processors require `name` to be set to either `minion` or `promethion` [...]"

Comment (avarsava) [reply README.md]: Done

Comment (avarsava) [reply README.md]: Done

**Pull request merged**